DAL/bodyparserApi.js

```diff
@@ -1,6 +1,6 @@
 
```

Comment:
This needs to be case insensitive. It is checking for upper case letters only in some places while http is only lowercase.

Reply:
Yep. forgot to do that while drafting the PR, cheers.